src/app/pages/LandingPage.svelte

```diff
@@ -1,56 +1,189 @@
+
+    color: var(--white);
+    background-position: bottom center;
+    background-repeat: no-repeat;
+    background-image: url(assets/wave.svg),
+      linear-gradient(
+        180deg,
+        var(--masthead-background-color) 0%,
+        var(--masthead-background-color) var(--masthead-content-height),
+        rgba(255, 255, 255, 0) var(--masthead-content-height)
+      );
+  }
+
+  .masthead__content {
+    display: flex;
+    width: 80vw;
+    max-width: var(--body-max-width);
+    margin: auto;
+
+    flex-flow: row nowrap;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .masthead__brand {
+    font-weight: bold;
+    color: var(--black, #111);
+  }
+
+  .masthead__title,
+  .masthead__description {
+    font-weight: 400;
+  }
+
+  .explanation {
+    margin: 2rem;
+
+    text-align: center;
+  }
+
+  .quick-start {
+    margin: auto;
+    min-width: 20em;
+    max-width: var(--quick-start-max-width);
+  }
+
+  .quick-start__step {
+    margin: 0 0 1rem;
+    border: 0;
+    padding: 0;
+
+    font-size: 1.25em;
+    line-height: 1.5;
+  }
+
+  .quick-start__submit {
+    margin: 2rem 0;
+    width: 100%;
+  }
+
+  .footer {
+    padding: 1rem;
+
+    color: var(--white, #fff);
+    background-color: var(--gray-dark);
+  }
+
+  .footer a:link,
+  .footer a:visited,
+  .footer a:active {
+    color: inherit;
+  }
+
+  .footer a:hover {
+    color: var(--gray-light);
+  }
+
+  .footer__copyright {
+    margin: auto;
+    max-width: var(--body-max-width);
+  }
+
+  .block {
+    display: block;
+
+    margin: 1rem auto 2rem;
+  }
+</style>
+
 <svelte:head>
   <title>Welcome to Predictive Text Studio</title>
+  <meta
+    name="description"
+    content="Add prediction and autocorrect to your language" />
 </svelte:head>
 
 <main>
-  <header>
-    <p role="doc-subtitle">Keyman</p>
-    <h1>Predictive Text Studio</h1>
-
-    <p>
-      Add
-      <strong>prediction</strong>
-      and
-      <strong>autocorrect</strong>
-      to your language
-    </p>
+  <header class="masthead">
+    <!-- wrapper div required to do margin: auto; trick -->
+    <div class="masthead__content">
+      <div class="masthead__copy">
+        <p class="masthead__brand" role="doc-subtitle">Keyman</p>
+        <h1 class="masthead__title">Predictive Text Studio</h1>
+
+        <p class="masthead__description">
+          Add
+          <strong>prediction</strong>
+          and
+          <strong>autocorrect</strong>
+          to your language
+        </p>
+      </div>
+      <img class="masthead__img" src="/assets/texting.svg" alt="" role="presentation">
+    </div>
   </header>
 
-  <section id="explanation">
+  <section class="explanation" id="explanation">
     <!-- TODO: figure that explains the flow -->
-
-    <a href="#get-started" class="button button--primary button--shadow">Get started</a>
+    <a href="#get-started">
+      <span class="button button--primary button--shadow button--large">Get started</span>
```

Comment:
`margin-bottom` here would also provide a little needed whitespace to match designs